osidb/tests/test_endpoints.py

```diff
@@ -1591,6 +1617,89 @@ def test_affect_delete(self, auth_client, test_api_uri):
         response = auth_client.get(affect_url)
         assert response.status_code == 404
 
+    def test_flawacknowledgment_create(self, auth_client, embargo_access, test_api_uri):
+        """
+        Test the creation of FlawAcknowledgment records via a REST API POST request.
+        """
+        # Source must be private in order for validation to pass.
+        flaw = FlawFactory(source=FlawSource.CUSTOMER)
+
+        flawacknowledgment_data = {
+            "name": "John Doe",
+            "affiliation": "Acme Corp.",
+            "from_upstream": False,
+            "flaw": str(flaw.uuid),
```

Comment:
Should be already included in the path.

Reply:
Fixed in commit 3c2d2dd147380fc8924fac4b9d539f85f86f36d4 (hopefully, see discussion thread https://github.com/RedHatProductSecurity/osidb/pull/265#discussion_r1250297359 )